docs/Data/external_hive_support.md

```diff
@@ -130,6 +130,188 @@ If you set the property **spark.sql.catalogImplementation = in-memory**, you can
 	
         snappy-sql> SELECT emp.EmployeeID, emp.FirstName, emp.LastName, o.OrderID, o.OrderDate FROM default.hive_employees emp JOIN snappy_orders o ON (emp.EmployeeID = o.EmployeeID) ORDER BY o.OrderDate;
 
+## Creating and Querying Hive Tables 
+
+To create or access, the hive tables, you must first create a database or schema in the external hive catalog from SnappyData. Else, the default database **default** is used.  Go to the Snappy Shell prompt and enter the following command:
+
+		snappy> create database hivedb;
+
+After creating the database or schema, you can create hive tables, load data into the hive tables, and run DDL queries on hive tables from SnappyData using one of the following methods:
+
+### Method 1
+
+In this method, you must use the schema or database name before the table name as shown in the following example:	
+
+    snappy> 
+    create table hivedb.hive_regions(RegionID int,RegionDescription string) row format delimited fields terminated by ',' tblproperties("skip.header.line.count"="1");
+
```

Comment:
I think it should be mentioned that DDLs will only be executed against the external hive if it uses hive-specific extensions in the CREATE TABLE (like "row format" above), or else specifically mention "using hive" after the table schema. In other words it is not possible to create Spark format EXTERNAL tables (e.g. a parquet table) in the external hive metastore.

Reply:
This change, and a few others related to documentation, are taken from another branch enterprise_docv1.2.0, so that master stays in sync with the latest.
Added your notes now.